### PR TITLE
Packaging

### LINKED ISSUE #39: Packaging with Poetry

### PR DESCRIPTION
**Pull request by dhananjaylatkar**
Closes #39

### CODE REVIEW

**Review by garuna-m6 — APPROVED** [review]
Not sure about the poetry lock file, rest lgtm

File: README.md [hunk 1]
```diff
@@ -1,183 +1,86 @@
...
 
-We recommend this if you never have used any other dependency manager, although if you have, feel free to switch to it. More on that below.
+### Install Poetry
 
-#### Creating the environment
-Create a virtual environment in the folder `.venv`.
+- macOS, Linux or WSL
 ```shell
-$ python -m venv .venv
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
```
Comment (Trisanu-007) [inline README.md:48]: Latest version of poetry can be installed with install-poetry.py instead with get-poetry.py. Check [here](https://python-poetry.org/docs/#updating-poetry)

File: README.md [hunk 2]
```diff
@@ -1,183 +1,86 @@
...
-$ source .venv/bin/activate.fish
-# Linux, Csh
-$ source .venv/bin/activate.csh
-# Linux, PowerShell Core
-$ .venv/bin/Activate.ps1
-# Windows, cmd.exe
-> .venv\Scripts\activate.bat
-# Windows, PowerShell
-> .venv\Scripts\Activate.ps1
+(Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python -
```
Comment (Trisanu-007) [inline README.md:53]: Same thing earlier with the installation.

**Review by Trisanu-007 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dhananjaylatkar) [reply README.md:53]: done

**Pull request merged**